src/gui/folderstatusmodel.cpp

```diff
@@ -934,8 +934,12 @@ void FolderStatusModel::slotSetProgress(const ProgressInfo &progress)
     if (totalSize > 0) {
         QString s1 = Utility::octetsToString( completedSize );
         QString s2 = Utility::octetsToString( totalSize );
+
         //: Example text: "5 minutes left, 12 MB of 345 MB, file 6 of 7"
```

Comment:
doesn't the translator comment need to be exactly one line above the tr(),  in any case, since there are two tr now, that might not be attached to the second tr which would still need a comment